### PR TITLE
Updated to 1.21.0

### PR DESCRIPTION
**Pull request by HydroGames-dev**

### CODE REVIEW

File: src/customiesdevs/customies/block/component/BreathabilityComponent.php
Comment (HydroGames-dev) [inline src/customiesdevs/customies/block/component/BreathabilityComponent.php:1]: [This](https://learn.microsoft.com/en-us/minecraft/creator/reference/content/blockreference/examples/blockcomponents/minecraftblock_breathability?view=minecraft-bedrock-stable)
or [This](https://github.com/MicrosoftDocs/minecraft-creator/blob/main/creator/Reference/Content/BlockReference/Examples/BlockComponents/minecraftBlock_breathability.md)

[the microsoft changes](https://github.com/MicrosoftDocs/minecraft-creator/commit/ba36f30f56f2e26cdc5b91eb10bc1f5046dc249b)

its not shown in bedrock dev wiki but its added in the game

File: src/customiesdevs/customies/block/component/UnitCubeComponent.php
Comment (HydroGames-dev) [inline src/customiesdevs/customies/block/component/UnitCubeComponent.php:1]: this is from https://bedrock.dev/docs/stable/Blocks

**Comment by HydroGames-dev** [comment]
`isProperty` in Items Components works different from components its kinda of the same

[Item Component](https://wiki.vg/Bedrock_Protocol#Item_Component)
tho its not needed i think `minecraft:component_name` is better i assume 
check it

**Comment by jeantkg** [comment]
> `isProperty` in Items Components works different from components its kinda of the same
> 
> [Item Component](https://wiki.vg/Bedrock_Protocol#Item_Component) tho its not needed i think `minecraft:component_name` is better i assume check it

`minecraft:component_name` is better but won't work so which is why we are using `isProperty`

**Comment by jeantkg** [comment]
I will merge the pr and do some changes myself

**Pull request merged**